src/main/java/pl/tomaszdziurko/jvm_bloggers/metadata/MetadataKeys.java

```diff
@@ -0,0 +1,10 @@
+package pl.tomaszdziurko.jvm_bloggers.metadata;
+
+public enum MetadataKeys {
```

Comment:
I followed by convention in the existing class, but I guess the better way is using constants :)